tools/IoTEdgeDevOps/VstsPipelineSync/db/dbo.VstsBug.sql

```diff
@@ -0,0 +1,17 @@
+/****** Object:  Table [dbo].[VstsBuildRun]    Script Date: 2/12/2020 4:56:48 PM ******/
+SET ANSI_NULLS ON
+GO
+
+SET QUOTED_IDENTIFIER ON
+GO
+
+CREATE TABLE [dbo].[VstsBug](
+	[QueryName] [varchar](20) NOT NULL,
+	[BugCount] [int] NOT NULL,
```

Comment:
you should create 2 more fields:
[InsertedAt] [datetime2](7) NOT NULL,
[UpdatedAt] [datetime2](7) NOT NULL,